pkg/controllers/networking/node_controller.go

```diff
@@ -86,6 +104,7 @@ func (r *NodeReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.
 
 	node.Labels[constants.LabelUnderlayNetworkAttachment] = attachedToString(underlayAttached)
 	node.Labels[constants.LabelOverlayNetworkAttachment] = attachedToString(overlayAttached)
+	node.Labels[constants.LabelBGPNetworkAttachment] = attachedToString(bgpAttached)
```

Comment:
is it better to patch `LabelBGPNetworkAttachment` only if global bgp network exist?

Reply:
Sorry, I misunderstood that such labels will be cleaned if the `attached` flag is false.

But I'm wondering if a node should be patched with a valued "false" attachment label if it's not be attached with an overlay/underlay/bgp network, maybe not patching or removing the no-attached label is the right behavior.

Reply:
There is a little different here, the true/false of underlay and overlay attachment label is always meaningful(correct), but if global bgp network not exists, the true/false of bpg attachment label will be **always** false, even this node belong to a normal underlay bgp network.
So i think it's better to remove this label if global bpg network not exists.

Reply:
In another words, "bgp attachment true/false label" is also meaningful (for literal meaning) if a bgp underlay network is attached or not, whether global bgp network not exists. We don't want it because it just seems useless and redundant if global bgp network is not exist.

I mean, for implementation, if labels seems useless and redundant (though it does have a  literal meaning), maybe we should treat them equally, which means, remove them.

Reply:
agreed，let's remove all the labels if it's false, because only true is meaningful for us.

Reply:
Done.